libs/external-router/src/transform/resolved-route-to-insertion-strategy.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffExternalRouterInsertionStrategy } from '../model/insertion-strategy.type';
+import { DaffExternallyResolvableUrl } from '../model/resolvable-route';
+import { DaffTypeRoutePair } from '../model/type-route-pair';
+
+/**
+ * Finds the inserter associated with the resolved route, if such an inserter exists.
+ */
+export const daffTransformResolvedRouteToInsertionStrategy = (
+  resolvedRoute: DaffExternallyResolvableUrl,
+  routeTypes: DaffTypeRoutePair[],
+): DaffExternalRouterInsertionStrategy => {
+  const routeType = routeTypes
```

Comment:
We already do a lookup very similar to this in another function, is there a way to merge these two lookup together so that we don't have to do this lookup twice?

Reply:
https://github.com/graycoreio/daffodil/blob/2d7ad2766bd0cafe9c15a3e20c7cb08c427b91bb/libs/external-router/src/transform/resolved-route-to-route.ts

Reply:
changed